### PR TITLE
P0001 clean

### PR DESCRIPTION
**Pull request by rakeshshrivas**

### CODE REVIEW

File: apps/proxy/src/app/layout/layout.component.ts [hunk 1]
```diff
@@ -56,13 +56,17 @@ export class LayoutComponent extends BaseComponent implements OnInit {
...
-        } else {
+        if (w <= breakpoint) {
             return true;
+        } else {
+            return false;
         }
     }
+
+    public toggleSideBarEvent(): void {
+        this.isSideNavOpen$.next((this.sideNavOpenState = !this.sideNavOpenState));
```
Comment (rahul1choudhary) [inline apps/proxy/src/app/layout/layout.component.ts:70]:
```suggestion
        this.isSideNavOpen$.next(!this.isSideNavOpen$.getValue());
```

File: apps/proxy/src/app/layout/layout.component.ts [hunk 2]
```diff
@@ -17,6 +17,8 @@ import { CookieService } from 'ngx-cookie-service';
 export class LayoutComponent extends BaseComponent implements OnInit {
     public toggleMenuSideBar: boolean;
     public logInData$: Observable<IFirebaseUserModel>;
+    public sideNavOpenState = true;
+    public isSideNavOpen$ = new BehaviorSubject<boolean>(this.sideNavOpenState);
```
Comment (rahul1choudhary) [inline apps/proxy/src/app/layout/layout.component.ts:21]:
```suggestion
    public isSideNavOpen$ = new BehaviorSubject<boolean>(true);
```

File: apps/proxy/src/app/layout/layout.component.ts [hunk 3]
```diff
@@ -17,6 +17,8 @@ import { CookieService } from 'ngx-cookie-service';
 export class LayoutComponent extends BaseComponent implements OnInit {
     public toggleMenuSideBar: boolean;
     public logInData$: Observable<IFirebaseUserModel>;
+    public sideNavOpenState = true;
```
Comment (rahul1choudhary) [inline apps/proxy/src/app/layout/layout.component.ts:20]:
```suggestion
```

**Review by rahul1choudhary — APPROVED** [review]

File: apps/proxy/src/app/layout/layout.component.ts (hunk 1 above)
Comment (rakeshshrivas) [reply apps/proxy/src/app/layout/layout.component.ts:70]: Done

File: apps/proxy/src/app/layout/layout.component.ts (hunk 2 above)
Comment (rakeshshrivas) [reply apps/proxy/src/app/layout/layout.component.ts:21]: Done

File: apps/proxy/src/app/layout/layout.component.ts (hunk 3 above)
Comment (rakeshshrivas) [reply apps/proxy/src/app/layout/layout.component.ts:20]: Done

File: apps/proxy/src/app/layout/layout.component.ts (hunk 2 above)
Comment (prateek-giddh) [reply apps/proxy/src/app/layout/layout.component.ts:21]: Only observable name should end with `$`, for a subject it should just be `isSideNavOpen`

**Review by prateek-giddh — APPROVED** [review]

**Review by rahul1choudhary — APPROVED** [review]

**Pull request merged**